clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/description/servergroup/ServerGroupParameters.groovy

```diff
@@ -106,7 +109,8 @@ class ServerGroupParameters {
       rawUserData: params.get('user_data'),
       tags: unescapePythonUnicodeJsonMap(params.get('tags') ?: null),
       sourceUserDataType: params.get('source_user_data_type'),
-      sourceUserData: params.get('source_user_data')
+      sourceUserData: params.get('source_user_data'),
+      asgResourceFilename: params.get('asg_resource_filename')
```

Comment:
I was following the naming convention already established and this is a template file name for the auto scaling resource.